### ISSUE TITLE
Since I updated date-fns to v2.0.0-alpha.32

### ISSUE THREAD

**Issue by trollepierre** [date-fns/date-fns#1199]
I got failed build, when updating to last version:
```
ERROR in static/js/vendors~app.da80acdb6f64b23af377.js from UglifyJs
Unexpected token: keyword «const» [static/js/vendors~app.da80acdb6f64b23af377.js:53646,0]
```
=>
https://github.com/trollepierre/recontact_travel_blog/pull/374

**Comment by kossnocorp** [comment]
Damn, I'm sorry! I'll fix that and issue another version.

**Comment by kossnocorp** [comment]
Fixed in `v2.0.0-alpha.33`
